lib/l10n/app_pl.arb

```diff
@@ -0,0 +1,3 @@
+{
+    "helloWorld": "You have pushed the button this many times:"
```

Comment:
In toPWR we stick with snacke_case naming convention, so maybe it's good to stick with it here as well :-)